packages/@webex/plugin-meetings/src/reachability/clusterReachability.ts

```diff
@@ -23,10 +21,32 @@ export type ClusterReachabilityResult = {
   xtls: TransportResult;
 };
 
+// data for the Events.resultReady event
+export type ResultEventData = {
+  protocol: 'udp' | 'tcp' | 'xtls';
+  result: 'reachable' | 'unreachable' | 'untested';
```

Comment:
Q. From where are we triggering the partial reachability event?

Reply:
I'm not sure what are you asking about, the `ResultEventData ` is used only with "resultReady" event and that event will actually always have result set to "reachable", because it's only emitted when we get an ICE candidate. I could change the type here to just say  `result: 'reachable'.
Or if you're asking about the partial reachability event that was mentioned on the confluence page, then that its emitted from [here](https://github.com/webex/webex-js-sdk/blob/7f573abc7ef45707c945ea9d3c4c9b59fc7b7f5f/packages/%40webex/plugin-meetings/src/reachability/index.ts#L685) but I've renamed the event and called it 'reachability:firstResultAvailable' instead of "reachability:partiallyDone". I feel it's a better name, once the PR is approved I'll update the confluence.